src/index.ts

```diff
@@ -17,6 +17,10 @@ export interface HtmlFileConfiguration {
     scriptLoading?: 'blocking' | 'defer' | 'module',
     favicon?: string,
     findRelatedOutputFiles?: boolean,
+    extraScripts?: (string | { 
```

Comment:
Any reason you chose 

```
{
  key: string,
  value: string,
}[]
```

over 

```
{
key1: value1,
key2, value2,
}
```

?

Reply:
I'm a little confused on this one... 
```ts
{
  key: string,
  value: string,
}[]
```
This is for the correct TypeScript typing?

Reply:
I think I explained this a bit confusing, so sorry for this. 😅 

I was wondering if `attrs` should just take a object instead of a list of  `{key: string, value: string}`.

Explicit, this would look like:

```js
{
  ...
  attrs: {
    "async": "",
    "type": "module",
  }
  ...
}
```

I think this looks cleaner then

```js
{
  ...
  attrs: [
    { key: "async", value: "" },
    { key: "type", value: "module"},
  ]
  ...
}
```

But I guess you chose this variant to support strings as well as objects?

Reply:
Oh gotcha, no problem. Added in latest commit.